### PR TITLE
New users list design + search + scrolling

### PR DESCRIPTION
**Pull request by clone1018**
image

### CODE REVIEW

File: lib/glimesh_web/uploaders/avatar.ex [hunk 1]
```diff
@@ -31,7 +31,8 @@ defmodule Glimesh.Avatar do
   end
 
   # Provide a default URL if there hasn't been a file uploaded
-  def default_url(_version, _scope) do
-    "/images/200x200.jpg"
+  def default_url(_version, scope) do
+    hash = :crypto.hash(:md5, String.downcase(scope.email)) |> Base.encode16(case: :lower)
+    "https://www.gravatar.com/avatar/#{hash}?s=200&d=robohash"
```
Comment (clone1018) [inline lib/glimesh_web/uploaders/avatar.ex:36]: Also implements Gravatar for users without avatars :)

File: lib/glimesh/accounts.ex [hunk 2]
```diff
@@ -11,7 +11,19 @@ defmodule Glimesh.Accounts do
   ## Database getters
 
   def list_users do
-    Repo.all(from u in User, where: not is_nil(u.confirmed_at))
+    Repo.all(from(u in User))
```
Comment (mja00) [inline lib/glimesh/accounts.ex:14]:
```suggestion
    Repo.all(from u in User, where: u.is_banned == false)
```
This'll make it skip over any globally banned users

File: lib/glimesh/accounts.ex [hunk 3]
```diff
@@ -11,7 +11,19 @@ defmodule Glimesh.Accounts do
...
+  end
+
+  def search_users(query, current_page, per_page) do
+    like = "%#{query}%"
+
+    Repo.all(
+      from u in User,
+        where: ilike(u.username, ^like),
```
Comment (mja00) [inline lib/glimesh/accounts.ex:22]:
```suggestion
        where: ilike(u.username, ^like),
        where: u.is_banned == false,
```

**Review by mja00 — CHANGES_REQUESTED** [review]
Just two things to ignore banned users from the user listing

**Pull request merged**